### DISCUSSION TITLE
分离存储市场问题汇总

### DISCUSSION THREAD

**Discussion by de-authority** [filecoin-project/community-china#149] [订单市场]
如果你有任何分离存储市场中遇到问题，可以发到下面留言，我们一起寻找解决方法，主要针对国内矿工，所以发到这里方便大家交流

**Comment by xiaoyuaiheshui** [comment]
lotus-miner info:
ERROR: fatal error calling 'Filecoin.MarketListIncompleteDeals': panic in rpc method 'Filecoin.MarketListIncompleteDeals': runtime error: invalid memory address or nil pointer dereference

**Reply by de-authority** [reply]
存储市场的API没有导入到miner节点

**Reply by de-authority** [reply]
在市场子系统里执行`lotus-miner --call-on-markets auth api-info --perm admin`然后export到miner节点里

**Reply by xiaoyuaiheshui** [reply]
哦哦，明白了。我看配置里面已经在做模块分离的事情，有没有考虑把windowpost的模块也拆出来呢？

**Reply by de-authority** [reply]
是的，会拆的，开发团队已经在做，逐步分离

**Reply by ChaiWenyu1983** [reply]
> 存储市场的API没有导入到miner节点

这个怎么导入？ 是设置环境变量吗？

**Reply by KaixuanZhao** [reply]
IPFS主网最大雅典娜矿池，RRmine global ，一站式采购托管运营，全部数据链上可查，全网算力领先1400PB，需要参与Datacap真实数据存储的朋友可以在小狐狸浏览器中复制打开https://www.rrmglobal.io/#/invite?invite_code=O3DKS2S0 欢迎加入社群 Wechet：kkmmjjnn00

**Comment by OliverRen** [comment]
这个文档只有配置,是否可以结合code一下啊.基于什么版本的源码.

**Reply by de-authority** [reply]
我当时测的时侯是1.11.2的devnet.　分离风险系数不高的。有问题你来。

**Comment by aizhikai** [comment]
vm      vm/vm.go:536    Send actor error        {"from": "f3rxgyvtotoyfmbdo6k7ouuwbtpk7b5bv5vg72eucpexknndetwb4h3rbinmpvhtvrtndptto3mrvchmznevuq", "to": "f01250863", "nonce": 7572, "method": "7", "height": "1144875", "error": "failed to submit proof for bulk verification (RetCode=7):\n    github.com/filecoin-project/specs-actors/v5/actors/builtin.RequireSuccess\n        /root/go/pkg/mod/github.com/filecoin-project/specs-actors/v5@v5.0.4/actors/builtin/shared.go:64"}

**Reply by de-authority** [reply]
这个报错好像与分离存储市场无关

**Comment by hxg8202** [comment]
worker 机上要设置  MARKETS_API_INFO 吗

**Reply by de-authority** [reply]
不需要的，但是如果你想要在worker上使用lotus-miner敲存储市场相关的命令，就需要导出这个环境变量。但大部分情况下，都习惯在miner敲这些命令。

**Comment by KaixuanZhao** [comment]
IPFS主网最大雅典娜矿池，提供一站式采购托管运营，fil存量数据迁移出海等服务，全网算力领先1400PB，需要B端1PB以上，C端50T，100T，200T（可扩容）和FIL存量数据迁移出海服务方案的滴滴我，欢迎加入社群Wechet：kkmmjjnn00

**Reply by de-authority** [reply]
请不要发送垃圾信息

**Comment by yunfens** [comment]
由于急用钱资金链断裂，项目整体转让，机房自建配置，3台空调，50千瓦发电机，30千瓦ups；存储配置：6台36盘14T存储机外加5台同配置空的存储机（不带硬盘），5台封装机，每天10T的封装能力，并且已经封装了430T，一台节点机，7张3090显卡，算力均为1t内存，16tp4510缓存盘，3090显卡adm7402，可以公司一起转，另可以免费提供技术支持！机房租金3000每月，可放置6台42U机柜，并可以扩充更多！

**Comment by zhaobo443** [comment]
FIL联合封装，用户提供质押币，公司提供设备和运维，收益60％,公司扣出20％服务费和20%设备费，合同期1080天，540天质押币退还，t3天封装。可实地考察。V:zhaobo9308

**Comment by ChaiWenyu1983** [comment]
我有个疑问，意思是miner 配置文件里的 Libp2p 都注释了，挪到 .lotusmarkets 的配置文件里吗？
